SQL/Archive/17.1/2017-03-24-examiner_decoupling_from_site.sql

```diff
@@ -0,0 +1,25 @@
+DROP TABLE IF EXISTS `examiners_psc_rel`;
+CREATE TABLE `examiners_psc_rel` (
+  `examinerID` int(10) unsigned NOT NULL,
+  `centerID` tinyint(2) unsigned NOT NULL,
+  `active` enum('Y','N') NOT NULL DEFAULT 'Y',
+  `pending_approval` enum('Y','N') NOT NULL DEFAULT 'N',
+  PRIMARY KEY  (`examinerID`,`centerID`),
+  KEY `FK_examiners_psc_rel_2` (`centerID`),
+  CONSTRAINT `FK_examiners_psc_rel_1` FOREIGN KEY (`examinerID`) REFERENCES `examiners` (`examinerID`) ON DELETE CASCADE ON UPDATE CASCADE,
+  CONSTRAINT `FK_examiners_psc_rel_2` FOREIGN KEY (`centerID`) REFERENCES `psc` (`CenterID`) ON DELETE CASCADE ON UPDATE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- ** WARNING: The next 2 lines will DROP column CenterID from the users table **
+-- ALTER TABLE examiners DROP foreign key FK_examiners_1;
+-- ALTER TABLE examiners DROP INDEX full_name;
+-- ALTER TABLE examiners DROP KEY FK_examiners_1;
+-- ALTER TABLE examiners DROP COLUMN `CenterID`;
+-- ALTER TABLE examiners DROP COLUMN `active`;
+-- ALTER TABLE examiners DROP COLUMN `pending_approval`;
+
+-- ALTER TABLE certification ADD CONSTRAINT `FK_certifcation_2` FOREIGN KEY (`examinerID`) REFERENCES `examiners` (`examinerID`);
+-- ALTER TABLE examiners ADD CONSTRAINT `U_examiners_1` UNIQUE KEY `full_name` (`full_name`);
+-- ALTER TABLE examiners MODIFY COLUMN `radiologist` tinyint(1) default 0;
```

Comment:
I don't like the idea of committing out SQL patch code, we should maybe add new infrastructure to better notify the users with this type of patches